test/.eslintrc.yaml

```diff
@@ -82,6 +82,8 @@ overrides:
       - es-module/*.mjs
       - parallel/*.js
       - parallel/*.mjs
+      - parallel/**/*.js
+      - parallel/**/*.mjs
```

Comment:
I think you can remove the old lines as the new ones should also match files without a subdirectory.